src/Microsoft.ML.TensorFlow/TensorFlow/Tensor.cs

```diff
@@ -848,9 +946,9 @@ private static unsafe object FetchJaggedArray(Type t, TFDataType dt, ref IntPtr
                             data += sizeof(Complex);
                             break;
                         case TFDataType.String:
-                            throw new NotImplementedException("String decoding not implemented for tensor vecotrs yet");
+                            throw Contracts.ExceptNotImpl("String decoding not implemented for tensor vecotrs yet");
```

Comment:
>vecotrs [](start = 102, length = 7)

vectors

Reply:
Across whole file

---
In reply to: [257011537](https://github.com/dotnet/machinelearning/pull/2546#discussion_r257011537) [](ancestors = 257011537)